runner/stats_coordinator.rb

```diff
@@ -53,43 +89,33 @@ def update_pod pod_id, data
       end
     end
 
-    def pod_try pod_name, time=nil
+    def installs time = nil
+      where = ''
+      where = "WHERE sent_at >= current_date - interval '#{time}'" if time
       query = <<-SQL
-        SELECT COUNT(dependency_name)
-        FROM install
-        WHERE dependency_name = $1
-        AND pod_try = true
+        SELECT dependency_name
+               count(CASE WHEN pod_try THEN 1 ELSE null END) AS try_true,
```

Comment:
Here you could use `pod_tries` and `downloads`, or `pod_try` and `download` for the aliases.